src/dsp_tools/commands/excel2xml/excel2xml_lib.py

```diff
@@ -136,38 +137,43 @@ def find_date_in_string(string: str) -> Optional[str]:
     # sanitize input, just in case that the method was called on an empty or N/A cell
     if not check_notna(string):
         return None
+    try:
+        return _find_date_in_string_throwing(string)
```

Comment:
I don't particularly like this pattern. In fact, no one of the called functions should raise an error, they should just re
turn None if they don't succeed. Then, we don't need a function with this strange name.

Reply:
I am with you on that, but I didn't want to make bigger changes in a simple linting PR. Feel free to adjust that in a separate PR and I will happily approve it :)